content/blog/2013-06-12-community-roundup.md

```diff
@@ -3,45 +3,46 @@ title: "Community Round-up #1"
 author: [vjeux]
 ---
 
-React was open sourced two weeks ago and it's time for a little round-up of what has been going on.
+El código de React fue hecho público hace dos semanas y ya es hora de un pequeño resumén de lo que ha estado sucediendo.
 
-## Khan Academy Question Editor {#khan-academy-question-editor}
+## Editor de preguntas de Khan Academy {#khan-academy-question-editor}
 
-It looks like [Sophie Alpert](http://sophiebits.com/) is the first person outside of Facebook and Instagram to push React code to production. We are very grateful for her contributions in form of pull requests, bug reports and presence on IRC ([#reactjs on Freenode](irc://chat.freenode.net/reactjs)). Sophie wrote about her experience using React:
+Parece que [Sophie Alpert](http://sophiebits.com/) es la primera persona externa de Facebook e Instagram en publicar código de React a producción. Estamos muy agradecidos por sus contribuciones en forma de _pull requests_, reportes de bugs y presencia en IRC ([#reactjs en Freenode](irc://chat.freenode.net/reactjs)). Sophie escribió sobre su experiencia usando React:
 
-> I just rewrote a 2000-line project in React and have now made a handful of pull requests to React. Everything about React I've seen so far seems really well thought-out and I'm proud to be the first non-FB/IG production user of React.
+> Acabo de reescribir un proyecto de 2000 líneas en React y varios _pull requests_ para React. Todo lo que he visto hasta ahora sobre React parece muy bien pensado y estoy orgullosa de ser la primera usuaria en producción de React que no es de FB/IG.
 >
-> The project that I rewrote in React (and am continuing to improve) is the Khan Academy question editor which content creators can use to enter questions and hints that will be presented to students:
+> El proyecto que reescribí en React (y sigo mejorando) es el editor de preguntas de Khan Academy que los creadores de contenido pueden usar para ingresar preguntas y sugerencias que se le presentarán a los estudiantes: 
+
 > <figure><a href="http://sophiebits.com/2013/06/09/using-react-to-speed-up-khan-academy.html"><img src="../images/blog/khan-academy-editor.png"></a></figure>
 >
-> [Read the full post...](http://sophiebits.com/2013/06/09/using-react-to-speed-up-khan-academy.html)
+> [Leer el post completo...](http://sophiebits.com/2013/06/09/using-react-to-speed-up-khan-academy.html)
 
-## Pimp my Backbone.View (by replacing it with React) {#pimp-my-backboneview-by-replacing-it-with-react}
+## Pimp my Backbone.View (reemplazandolo con React) {#pimp-my-backboneview-by-replacing-it-with-react}
```

Comment:
Hey @mishelashala, you'll still need to translate this.

Reply:
Also, it should be `reemplazándolo` instead of `reemplazandolo`

Reply:
@alejandronanez Isn't that a hyperlink? I think those goes in English since @tesseralis left all those in english sometime ago 🤔

Reply:
The part in `{# ... }` doesn't need to be translated but "Pimp my Backbone.View" might need to be :P It's a reference to [a show](https://es.wikipedia.org/wiki/Pimp_my_ride) so I don't know what the best way to translate that would be

Reply:
_Enchúlame el Backbone.View_ sounds weird.

Reply:
_Mejorame el Backbone.View_? or just _Mejora el Backbone.View_?

Reply:
_Mejora el Backbone.View_ sounds about right!